apps/rand.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 1998-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1998-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Copyright.  I'll stop commenting on that.  Please undo the changes if you didn't touch the file.